app/views/catalog/_home_text.html.erb

```diff
@@ -5,18 +5,21 @@
 
 <%= render partial: 'home_new_additions', locals: {
   type: 'duke_authors',
+  scope: 'catalog',
```

Comment:
There are URL helpers that should set the right URL for various application functions based on context: `search_action_url(add_facet_params(field, value))`

Reply:
@corylown -- Excellent, thanks!

Reply:
@corylown I've refactored the scope to use `search_action_url` in a new commit -- just pushed up. Most of the logic lives in JS (in `new_additions.js`), so it's a little tricky, but I've revised how that JS assembles URLs for the AJAX GETs to catalog.json vs. trln.json endpoints and also the item URLs.

Reply:
Those changes are here: 
https://github.com/duke-libraries/dul-argon-skin/pull/177/commits/298a77d2090349a2a93d62f574c3597d6e5cbea0